### PR TITLE
Revise Building

### PR DESCRIPTION
**Pull request by mforets**

### CODE REVIEW

**Comment by mforets** [comment]
For reference, the script from last year is [in the ARCH2018_RE repo](https://github.com/JuliaReach/ARCH2018_RE/blob/master/src/Building/build.jl).

I've updated it to the latest changes in Reachability, and made some cleanup. Checking the runtimes now.

File: models/ARCH/AFF/Building/building.jl [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+# Load model
+# ==============================
+file = @relpath "SX/Building_more_decimals.xml"
+H = readsxmodel(file, ST=ConstrainedLinearControlContinuousSystem)
+
+# ================
+# BLDF01 - BDS01
+# ================
+
+n = size(H.modes[1].A, 1)-1 # the sx model has "time" as a state varaible
```
Comment (schillic) [inline models/ARCH/AFF/Building/building.jl]:
```suggestion
n = size(H.modes[1].A, 1)-1 # the sx model has "time" as a state variable
```

**Comment by mforets** [comment]
```julia
julia> include("models/ARCH/AFF/Building/building_benchmark.jl")
Computing successors 100%|██████████████████████████████| Time: 0:00:03
Computing successors 100%|██████████████████████████████| Time: 0:00:03
Computing successors 100%|██████████████████████████████| Time: 0:00:03
Computing successors 100%|██████████████████████████████| Time: 0:00:03
Computing successors 100%|██████████████████████████████| Time: 0:00:02
Computing successors 100%|██████████████████████████████| Time: 0:00:03
Computing successors 100%|██████████████████████████████| Time: 0:00:02
Computing successors 100%|██████████████████████████████| Time: 0:00:02

Computing successors 100%|██████████████████████████████| Time: 0:00:03
Computing successors 100%|██████████████████████████████| Time: 0:00:02
Computing successors 100%|██████████████████████████████| Time: 0:00:02
Computing successors 100%|██████████████████████████████| Time: 0:00:03
Computing successors 100%|██████████████████████████████| Time: 0:00:02
Computing successors 100%|██████████████████████████████| Time: 0:00:03
Computing successors 100%|██████████████████████████████| Time: 0:00:02
Computing successors 100%|██████████████████████████████| Time: 0:00:02
Computing successors 100%|██████████████████████████████| Time: 0:00:03
Computing successors 100%|██████████████████████████████| Time: 0:00:03
Computing successors 100%|██████████████████████████████| Time: 0:00:03
Computing successors 100%|██████████████████████████████| Time: 0:00:03
(1/1) benchmarking "Build"...
  (1/4) benchmarking ("BLDF01-BDS01", "dense")...
Computing successors 100%|██████████████████████████████| Time: 0:00:02
Computing successors 100%|██████████████████████████████| Time: 0:00:02
Computing successors 100%|██████████████████████████████| Time: 0:00:02
  done (took 8.000010119 seconds)
  (2/4) benchmarking ("BLDC01-BDS01", "dense")...
Computing successors 100%|██████████████████████████████| Time: 0:00:02
Computing successors 100%|██████████████████████████████| Time: 0:00:03
  done (took 5.650217819 seconds)
  (3/4) benchmarking ("BLDF01-BDS01", "discrete")...
Computing successors 100%|██████████████████████████████| Time: 0:00:02
Computing successors 100%|██████████████████████████████| Time: 0:00:02
  done (took 5.592449223 seconds)
  (4/4) benchmarking ("BLDC01-BDS01", "discrete")...
Computing successors 100%|██████████████████████████████| Time: 0:00:03
Computing successors 100%|██████████████████████████████| Time: 0:00:02
  done (took 5.682374324 seconds)
done (took 25.538641747 seconds)
1-element BenchmarkTools.BenchmarkGroup:
  tags: []
  "Build" => 4-element BenchmarkTools.BenchmarkGroup:
          tags: []
          ("BLDF01-BDS01", "dense") => TrialEstimate(2.437 s)
          ("BLDC01-BDS01", "dense") => TrialEstimate(2.470 s)
          ("BLDF01-BDS01", "discrete") => TrialEstimate(2.489 s)
          ("BLDC01-BDS01", "discrete") => TrialEstimate(2.497 s)
1-element BenchmarkTools.BenchmarkGroup:
  tags: []
  "Build" => 4-element BenchmarkTools.BenchmarkGroup:
          tags: []
          ("BLDF01-BDS01", "dense") => TrialEstimate(2.474 s)
          ("BLDC01-BDS01", "dense") => TrialEstimate(2.522 s)
          ("BLDF01-BDS01", "discrete") => TrialEstimate(2.502 s)
          ("BLDC01-BDS01", "discrete") => TrialEstimate(2.535 s)
```

**Comment by schillic** [comment]
Did you forget to add the benchmark file?

**Comment by mforets** [comment]
I did.

**Pull request merged**